lessons/09-clientside-frameworks/README.md

```diff
@@ -1,33 +1,33 @@
 #Lesson 9 - Client-side Javascript Frameworks
 
-##Intro
+##Introduction
 So far we have seen how to organize an increasingly complex system of components on our node server. 
-We will now take a side step to discuss another organizational system in which instead of having lot of structure on the server, we have that structure on the client. 
+We will now take a side step to discuss another organizational system in which instead of having lot of structure on the **server**, we have that structure on the **client**. 
```

Comment:
Using front-end frameworks doesn't necessarily move complexity/structure from the server to the client, but it provides a structure (a "framework") for organizing and manipulating the user interface